config/emoji.php

```diff
@@ -23,4 +23,38 @@
 
     'token' => env('GITHUB_TOKEN', null),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Cache Connection
+    |--------------------------------------------------------------------------
+    |
+    | Here you may tell us what cache connection to use. To use the default,
+    | set this to null.
+    |
+    */
+
+    'connection' => null,
```

Comment:
Shall we use `CACHE_DRIVER` on this?

Reply:
Nah. Setting to null means when we call `->store()` on the cache driver, we really do get the default. if we were to use env in the config file, we're making assumptions about the way the user has setup their cache config.